README.md

```diff
@@ -140,11 +139,11 @@ When this happens, it may create a large number of junk tables in the catalog.
 
     The table at `s3://some-data-bucket/table-path/another-table/` will be deleted.
 
--   A table with the same location as another, with a name that's a superstring of the other's (this is from the glue crawler semantic of creating tables which would otherwise have the same name with the name {table}-somelongid).
+-   A table with the same location as another, with a name that's a superstring of the other's (this is from the Glue Crawler semantic of creating tables which would otherwise have the same name with the name {table}-somelongid).
 
     For example, if you have the tables `foo` and `foo-buzzer`, both with the same location, `foo-buzzer` will be deleted.
 
 ## Running `create-partitions` as a Lambda
 
-Journera's biggest use for this library is as a Glue Crawler replacement for tables and datasets the glue crawlers have problems parsing.
+Journera's biggest use for this library is as a Glue Crawler replacement for tables and datasets the Glue Crawlers have problems parsing.
```

Comment:
This seems redundant with the Background section.

Reply:
I think this is more pointing to the fact that the lambda docs are elsewhere which is a useful thing for a contributor/user to know.

Reply:
I think this should stay. It may be redundant but I don't see that as a problem because people may come to this section without reading the background section.